src/contact/components/EditInfo.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+
+export default class EditInfo extends React.Component {
+	constructor(props){
+		super(props);
+	}
+	componentDidMount(){
+		if(this.props.info === null){
```

Comment:
What is the purpose of this else if conditions? this componentDidMount only fires once in app lifecycle

Reply:
This is for retaining change in data. If data is changed then component won't initialize with new data. It will show modified data.

Reply:
ok